flow/error_definitions.h

```diff
@@ -197,6 +197,7 @@ ERROR( client_lib_already_exists, 2119, "Client library with same identifier alr
 ERROR( client_lib_not_found, 2120, "Client library for the given identifier not found." )
 ERROR( client_lib_not_available, 2121, "Client library exists, but is not available for download." )
 ERROR( client_lib_invalid_binary, 2122, "Invalid client library binary." )
+ERROR( dsa_error, 2123, "Digital signature algorithm error" )
```

Comment:
Please make this code `6002` -- we have all AuthZ error codes in 6000

Reply:
Done!